### PR TITLE
index.md update

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,49 +1,29 @@
-<img src="images/logo.svg" alt="logo" width="200"/>
+<img src="icon-probabilistic.png" alt="logo" width="200"/>
```
Comment (K-Jo) [inline docs/index.md]: this is removing the wrong line.

File: docs/index.md [hunk 2]
```diff
@@ -1,49 +1,29 @@
...
 
 # RedisBloom - Probablistic Datatypes Module for Redis
 
-This module provides three datatypes, a Scalable **Bloom Filter**, **Cuckoo Filter** and a **Count-Mins-Sketch**.
-The first two datatypes are used to determine (with a given degree of certainty) whether
-an item is present or absent from a collection while the last is used to count the
-frequency of the different items in sub-linear space.
-
+RedisBloom module provides four datatypes, a Scalable **Bloom Filter** and **Cuckoo Filter**, a **Count-Mins-Sketch** and a **Top-K**.
+**Bloom and Cuckoo filters** are used to determine (with a given degree of certainty) whether an item is present or absent from a collection. While **Count-Min Sketch** is used to count the frequency of the elephant items in sub-linear space and **Top-K** maintains a list of K most frequent items.
```
Comment (K-Jo) [inline docs/index.md]: I believe the explanation of count-min sketch focuses on the implementation where the others focus on explain what they enable.

File: docs/index.md [hunk 3]
```diff
@@ -1,49 +1,29 @@
...
+1. [Clients](#clients)
+1. [Webinars](#webinars)
+1. [Past blog posts](#past-blog-posts)
 
 Note: You can also [build and load the module](#building-and-loading-redisbloom) yourself.
 
-You can find a command reference in [Bloom Commands.md](Bloom_Commands.md) and
-[Cuckoo Commands](Cuckoo_Commands.md)
-
+## Command references
```
Comment (K-Jo) [inline docs/index.md]: I would just point to the documentation: redisbloom.io

File: docs/index.md [hunk 4]
```diff
@@ -77,7 +56,20 @@ $ redis-server --loadmodule /path/to/redisbloom.so INITIAL_SIZE 400 ERROR_RATE 0
 The default error rate is `0.01` and the default initial capacity is `100`.
 
 ## Bloom vs. Cuckoo
-
 Bloom Filters typically exhibit better performance and scalability when inserting
 items (so if you're often adding items to your dataset then Bloom may be ideal),
 whereas Cuckoo Filters are quicker on check operations and also allow deletions.
+
+## Clients
```
Comment (K-Jo) [inline docs/index.md]: can you use the same structure as we use for redisgraph.io?  A table with license and author?

Comment (ashtul) [reply docs/index.md]: :+1:

Comment (ashtul) [reply docs/index.md]: Love it

**Comment by ashtul** [comment]
@K-Jo, should be fine now

**Comment by ashtul** [comment]
Can view updated file here
https://github.com/RedisBloom/RedisBloom/blob/55ec675334d81f02f95353c69b97e7a3940b8d04/docs/index.md

**Pull request merged**